platform.md

```diff
@@ -1,10 +1,9 @@
 # Platform Interface Specification
 
-This document specifies the interface between a lifecycle and a platform.
+This document specifies how to process buildpack lifecycle steps to ensure compatibility
+between end-user tools.
```

Comment:
```suggestion
This document specifies the interface between a lifecycle program and a platform.
```
We can add "program," but this should cover the components we're defining an interface between:

Platform spec:
(platform <-> lifecycle) <-> buildpacks

Buildpack spec:
platform <-> (lifecycle <-> buildpacks)

Reply:
I didn't know there is a difference here. Maybe there are already some user stories about these differences? Stories that we could have started with in the first place to clarify problems that this design is meant to solve?